### ISSUE TITLE
This app can not launch both on my pc and tablet anymore.

### ISSUE THREAD

**Issue by JingeOnline** [microsoft/fluent-xaml-theme-editor#44]
I download this app from Microsoft Store.
In this week, I find the app can not launch anymore, either on pc or tablet, just stuck in the Logo page. 
![image](https://user-images.githubusercontent.com/42198086/83669708-580d1d80-a615-11ea-91d2-305586d1f43e.png)
It used to be work well. I don't know the reason.

My Pc OS version: 19041.264
My Tablet OS version: 18363.836

If you need more information, please let me know.

**Comment by kikisaints** [comment]
Hey! I'm sorry you ran into this, I keep trying to track this bug down but I'm at a loss as to what's causing it. I think it's a localization error but I cannot get it to repro.

**Comment by JingeOnline** [comment]
Yes, you are right. I followed your guide and change the system language to "English". The app work again. 
Then I turn it back to "中文（chinese simplified）". It stilll stuck in loge page.
Anyway, thanks for your help, I can use this app now. And hope you find the bug soon.

**Comment by massijay** [comment]
I have this problem too with store version, works fine if I compile it from this repo.
My system language is italian.
I noticed that store version has a greater version number than this repo's, maybe this repo is not updated with latest code?

**Comment by loics2** [comment]
Same, with system language set tot french and works great when compiled from source.

**Comment by LuanVSO** [comment]
same here, system language is set to Portuguese (brasilian)

**Comment by jfranki** [comment]
I just gave the app a 1 star rating in the store for this (my OS is in Spanish and it gets stuck in the splash screen).
I can't believe they haven't patched it yet, so I assume it's abandoned, in which case they should pull it from the store.
